src/services/api.js

```diff
@@ -1,7 +1,12 @@
 export async function getCategories() {
-  // Começando !!
+  const endPoint = 'https://api.mercadolibre.com/sites/MLB/categories';
+  const response = await fetch(endPoint);
+  return  await response.json();
```

Comment:
Acho que tem um espaço a mais aqui entre o `return` e o `await`.
```suggestion
  return await response.json();
```